github/PullRequest.py

```diff
@@ -488,26 +496,34 @@ def is_merged(self):
         :calls: `GET /repos/:owner/:repo/pulls/:number/merge <http://developer.github.com/v3/pulls>`_
         :rtype: bool
         """
+        github._api_preview = True
```

Comment:
I was wondering about this -- here you set the _api_preview to True; overriding what the user provided in the Github constructor. I'm wondering if that will break the merge method for folks who might be using an older Github Enterprise that does not support the new functionality. The approach I took in https://github.com/makuk66/PyGithub/commit/e1546d48ee4935e5a5b50addc1b9a4811abe5c6f#diff-f0e149166d9b01e4a3e347716d866179R513 was to check if preview was on, and if not, abort if the new features are being used. Thoughts on what is better?